data/city_data.rb

```diff
@@ -0,0 +1,23 @@
+class CityData
+
+  attr_reader :city_short_form
```

Comment:
Best to put under `private` so it's accessible only within the class

Reply:
Noted, thank you.